frontend/src/scenes/surveys/surveysLogic.tsx

```diff
@@ -19,6 +19,7 @@ export enum SurveysTabs {
     Archived = 'archived',
     Notifications = 'notifications',
     History = 'history',
+    Settings = 'settings',
```

Comment:
It's confusing to me as a user that the survey customization option is called `customization` in the Survey editor but `Settings` as the top-level config item.  

In my opinion, the section in the survey editor should be called `Appearance`.  Not sure what to call this section, though.  I guess `Settings` is fine for now.  Will there be more than just the appearance configuration option in this section going forward?

Reply:
Yeah, the plan is to have templates in the future, but that can go in a different tab.